composer.json

```diff
@@ -57,14 +57,16 @@
         "phpcs": "phpcs . -s",
         "phpcbf": "phpcbf . -p -s",
         "phpstan": "phpstan analyse --memory-limit 512M .",
+        "phpstan-next": "phpstan analyse --level 6 --memory-limit 512M $(find . -type d -name 'vendor' -prune -o -name '*.php' -o -name '*.phtml' | grep -Fxvf ./tests/phpstan-next.txt | sort | paste -s)",
```

Comment:
This command finds all relevant `*.php` and `*.phtml` files and then remove those that are in `phpstan-next.txt` before calling PHPStan on the remaining ones.

Reply:
Example: https://github.com/FreshRSS/FreshRSS/pull/5230/commits/a5cc7b20f6b05ceae73a90dc418345f7f0dede6f